baggage/baggage_test.go

```diff
@@ -205,9 +208,9 @@ func TestNewBaggageWithDuplicates(t *testing.T) {
 	assert.Equal(t, want, b)
 }
 
-func TestNewBaggageErrorInvalidMember(t *testing.T) {
-	_, err := New(Member{key: ""})
-	assert.ErrorIs(t, err, errInvalidKey)
+func TestNewBaggageErrorEmptyMember(t *testing.T) {
+	_, err := New(Member{isEmpty: true})
+	assert.ErrorIs(t, err, errInvalidMember)
```

Comment:
You have removed a valid test.
Creating a Baggage with a `Member{}`.  It doesn't necessarily need to return an error, but it should be an empty baggage.
There is also the test case where a `Member{}` is in the list, it should return baggage with that Member removed.

Reply:
Good catch. I've renamed `isEmpty` to `hasData` so an empty struct is really marked as invalid, and brought back that test.